armi/tests/armiRun.yaml

```diff
@@ -51,6 +53,7 @@ settings:
   percentNaReduction: 10.0
   power: 100000000.0
   refType: inner igniter fuel
+  reloadDBName: reloadingDB.h5
```

Comment:
Interesting. Why did this get added here?

Reply:
This needed to be added so that a reload file would actually be loaded up in the [`test_prepRestartRun`](https://github.com/terrapower/armi/pull/637/files#diff-aca297e341a545c450119034f3293962547c6d11999bd09426d5504c9b241d4fR131). Before, the test that was in place didn't require actually loading anything because it only checked against the case settings. But that was no longer possible, so an actual restart file needed to be loaded, and thus the case setting pointing to that restart file needed to be added.